### PR TITLE
feat(project): Add firefox android beta task id tracking for external integration testing.

### LINKED ISSUE #11420: Add fenix beta task id for tracking
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-4820)

### PR DESCRIPTION
**Pull request by b4handjr**
Because

- We want to run certain jobs based on external dependency changes as they happen.

This commit

- Adds firefox for android (fenix) nightly builds to our key store for us to track and run tests against.
- Also renamed the old `fenix` key so it properly represents what they key is tracking.

Fixes #11420

### CODE REVIEW

File: scripts/update-firefox-keys.sh [hunk 1]
```diff
@@ -15,12 +15,19 @@ fi
...
+        echo FIREFOX_FENIX_BETA_TASK_ID TASK ID "${TASK_ID}"
+        echo "FIREFOX_FENIX_BETA_TASK_ID=${TASK_ID}" > firefox-fenix-beta-build.env
+        mv firefox-fenix-beta-build.env experimenter/tests
+        ;;
+    fenix-nightly)
+        INDEX_BASE="mobile.v3.firefox-android.apks.fenix-nightly.latest"
+        TASK_ID=$(curl ${CURLFLAGS[@]} "${TASKCLUSTER_API}/tasks/${INDEX_BASE}" | jq '.tasks[] | select(.namespace == "mobile.v3.firefox-android.apks.fenix-nightly.latest.x86_64") | .taskId')
+        echo FIREFOX_FENIX_NIGHTLY_TASK_ID TASK ID "${TASK_ID}"
+        echo "FIREFOX_FENIX_NIGHTLY_TASK_ID=${TASK_ID}" > firefox-fenix-nightly-build.env
+        mv firefox-fenix-nightly-build.env experimenter/tests
```
Comment (jaredlockhart) [inline scripts/update-firefox-keys.sh:30]: Isn't there some way to parameterize this rather than write it twice? 🤔

Comment (b4handjr) [reply scripts/update-firefox-keys.sh:30]: 🙃

**Review by jaredlockhart — APPROVED** [review]
Yep this is good, if you wanna parameterize that script I'd say do, otherwise let's just get this landed and keep adding more channels 👍 

Thnx @b4handjr! 🎉

**Review by jaredlockhart — APPROVED** [review]
👏 👏 👏 👏 👏 

Yeah this reads much better, thank you 🙏 🙏 🙏 🙏 🙏

**Review by jaredlockhart — APPROVED** [review]
Looks good! 💯 💯 💯

**Pull request merged**